lib/backend/k8s/conversion/conversion.go

```diff
@@ -555,7 +570,78 @@ func (c Converter) SplitNetworkPolicyRevision(rev string) (crdNPRev string, k8sN
 		return
 	}
 
-	crdNPRev = revs[0]
-	k8sNPRev = revs[1]
+	RevA = revs[0]
+	RevB = revs[1]
+	return
+}
+
+// serviceAccountNameToProfileName creates a profile name that is a join
+// of 'ksa/' + namespace + "/" + serviceaccount name.
+// namespace can only have [a-z0-9] and '-'. But serviceaccount name may have
+// '.'.
```

Comment:
I don't think it matters that the SA can have a dot since it is the last section in the name.